crates/bevy_ecs/src/entity/map_entities.rs

```diff
@@ -122,4 +171,73 @@ impl EntityMap {
     pub fn iter(&self) -> impl Iterator<Item = (Entity, Entity)> + '_ {
         self.map.iter().map(|(from, to)| (*from, *to))
     }
+
+    /// Creates an [`EntityMapper`] from this [`EntityMap`] and scoped to the provided [`World`], then calls the
+    /// provided function with it. This allows one to allocate new entity references in the provided `World` that are
+    /// guaranteed to never point at a living entity now or in the future. This functionality is useful for safely
+    /// mapping entity identifiers that point at entities outside the source world.
+    ///
+    /// # Arguments
+    ///
+    /// * `world` - The world the entities are being mapped into.
+    /// * `f` - A function to call within the scope of the passed world. Its return value is then returned from
+    ///         `world_scope` as the generic type parameter `R`.
```

Comment:
I would add this description to the text above and remove the whole `Arguments` section.